### PR TITLE
Remove BatchNormalization layer.

### PR DESCRIPTION
**Pull request by hgaiser**
Forgot to remove the BN layer itself :)

Also, remember that removing this means that models saved with the custom BN layer won't work with the latest version of keras-resnet because it will try to initialize the `BatchNormalization` layer with a `freeze` argument, which doesn't exist in Tensorflow. Easiest is to create a new model of the same number of classes, load the weights into that model and then save it again.

### CODE REVIEW

**Comment by vcarpani** [comment]
LGTM, send an email around the vision team and then merge ;)
